extensions/simctl/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "simctl",
```

Comment:
Is the term `simctl` known? Maybe it's just me, but I wonder if having the `name` as `Simulator Control` is clearer.

Reply:
@peduarte 
Apparently, the package.json's name has recommendation role.
In read this, It's better "simulator-control" than that. right?

https://docs.npmjs.com/cli/v9/configuring-npm/package-json#name

Reply:
Yeah `simulator-control` would be good I think!